tumblr-gif-search/flow-index.yml

```diff
@@ -21,8 +21,7 @@ pods:
     separated_workspace: true
```

Comment:
remember separatrd workspace is deprcated

Reply:
Removed